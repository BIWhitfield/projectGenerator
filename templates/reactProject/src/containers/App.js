import React, { Component } from 'react';

if (process.env.NODE_ENV !== 'production') {
  console.log('Looks like we are in development mode!'); // eslint-disable-line
}

class App extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  componentDidMount() {
    console.log('AND AWAY. WE. GO.');
  }

  render() {
    return (
      <div>
        <h1>This is my new React App</h1>
      </div>
    );
  }
}

export default App;
