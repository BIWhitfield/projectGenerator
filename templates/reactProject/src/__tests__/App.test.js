import React from 'react';
import renderer from 'react-test-renderer';
import App from '../containers/App';

describe('App Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
