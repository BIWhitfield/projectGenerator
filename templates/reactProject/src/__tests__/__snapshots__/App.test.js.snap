// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders correctly 1`] = `
<div>
  <h1>
    This is my new React App
  </h1>
</div>
`;
